src/Pods/Whatsit/Block.php

```diff
@@ -43,18 +43,78 @@ public function get_block_args() {
 			'category'        => $category,
 			'icon'            => $this->get_arg( 'icon', 'align-right' ),
 			'keywords'        => Tribe__Utils__Array::list_to_array( $this->get_arg( 'keywords', 'pods' ) ),
-			'supports'        => $this->get_arg( 'supports', [
-				'html' => false,
-			] ),
+			'supports'        => $this->get_arg( 'supports', [] ),
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 7ff04a2b62ea717f83ce83c7e7274519</sub>